Makefile

```diff
@@ -1,19 +1,43 @@
 # holochain Makefile
 
+F=RUSTFLAGS="-Dwarnings"
+
 # mark everything as phony because it doesn't represent a file-system output
-.PHONY: default build-workspace test-workspace
+.PHONY: default \
+	static-all static-fmt static-toml static-clippy static-doc \
+	build-workspace test-workspace
 
 # default to running everything (first rule)
 default: build-workspace test-workspace
 
+# execute all static code validation
+static-all: static-fmt static-toml static-clippy static-doc
+
+# ensure committed code is formatted properly
+static-fmt:
+	$(F) cargo fmt --check
+
+# lint our toml files
+static-toml:
+	cargo install taplo-cli
```

Comment:
Would slightly prefer a pinned version here and we can update when we want - although this tool does seem fairly stable so maybe not likely to change its opinion on formatting rules